src/components/SecondaryNav/SecondaryNav.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+/**
+ * Secondary navigation typically used under the header
+ *
+ * Must pass an array of NavLinks for the navigation bar.
+ * Each NavLink should contain an anchor with appropriate href, or a react
+ * router Link or equivelant.
+ */
+const SecondaryNav = ({ buttons, className }) => {
+  const navClass = classes('SecondaryNav', className);
+  return (
+    <nav className={navClass}>
+      <ul>
```

Comment:
should be a class here so that users can override styles easily and semantically